CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ set (Seastar_DPDK_MACHINE
 
 project (Seastar
   VERSION 1.0
+  DESCRIPTION "Event driven asynchronous programming framework"
+  HOMEPAGE_URL "http://seastar.io/"
```

Comment:
nit, strictly speaking, this is not part of this change.

Reply:
Fixed.